### PR TITLE
Postgres refactor

### PR DESCRIPTION
**Pull request by theofficetroll**
Stuff was getting mangled through spaghettification. I may move these back into a separate file later, but first I'll need to untangle the spaghetti and remove the unnecessary steps.

https://trello.com/c/6gqyhTpN

### CODE REVIEW

File: postgresdb/postgres.js [hunk 1]
```diff
@@ -5,68 +5,14 @@ const pool = new Pool (products);
...
-  })
-}
-
 module.exports = {
-  get,
   getAll,
-  newEntry,
-  newTable,
-  remove,
-  update,
```
Comment (skorpion-g) [inline postgresdb/postgres.js:71]: I see there were a lot of not useful postgres commands? Were these commands written by you earlier that you deemed unnecessary?

Comment (skorpion-g) [reply postgresdb/postgres.js:71]: Nevermind, you moved them to a diff location and updated them to PostGres. I assume these were initially mySQL

File: seedGenerator.js [hunk 2]
```diff
@@ -5,25 +5,30 @@ const fs = require('fs');
...
           category: generator.getCategory(),
-          style: style,
-        })
+          style: 1,
+        }
+        writer.write(document);
+        if (!writer.write(document)) {
+          await new Promise((resolve) => writer.once("drain", resolve));
+        }
+        ++i;
```
Comment (skorpion-g) [inline seedGenerator.js:26]: ah, either you're a man of culture and are implementing some of the fun ++ operands before "i", I think "i += 1" is also syntactically the same.

File: server/index.js [hunk 3]
```diff
@@ -37,9 +41,65 @@ app.use(parser.json());
...
-app.use('/', router);
+
+// Moved CRUD here because data was getting mangled
+app.get('/product/:id', async(req, res) => {
+  try {
+    const { id } = req.params;
+    const data = await pool.query('SELECT * FROM products WHERE productId = $1', [id]);
+
+    if (!data) {
+      return res.status(400).send('No such product');
```
Comment (skorpion-g) [inline server/index.js:52]: as simple as it is, this error msg will probably make Leslie sleep happier at night

**Review by skorpion-g — APPROVED** [review]
I see this was primarily a slight refactor for PostGres, Good stuff!

File: seedGenerator.js (hunk 2 above)
Comment (theofficetroll) [reply seedGenerator.js:26]: I tend to go with i++ in most cases, but I ran into some odd issues with timing that doing this form worked better with.

File: server/index.js (hunk 3 above)
Comment (theofficetroll) [reply server/index.js:52]: And Nu Ralik knows Leslie could use more sleep!

**Pull request merged**